TEMPLATES/mega-linter.yml

```diff
@@ -120,7 +120,8 @@ jobs:
             (
               github.event_name == 'push' ||
               github.event.pull_request.head.repo.full_name == github.repository
-            )
+            ) &&
+            !contains(github.event.head_commit.message, 'skip fix')
```

Comment:
NOT in https://github.com/oxsecurity/megalinter/pull/3032/files#diff-2a390738250fd23bc48f50e25700e0b2f06b62072aef23258772b1897a9b431bR271-R284

Reply:
@rasa please propose a change for this in #3032 and I will accept.

Reply:
I would prefer the `||` solution, but it's more important that we apply this logic across all five (?) instances of *mega-linter*.yml.

Reply:
> ... but it's more important that we apply this logic across all five (?) instances of *mega-linter*.yml.

Seems weird like that. We have one that is our own for dogfooding (the python@beta), another one for the second project (the node mega-linter-runner) again for dogfooding, then there is the one that is for the template for users, and all the duplicatas built into the documentation... I expect that these update from their sources when running `./build.sh`, except if I'm wrong (and we should fix it).